Mlem/Models/Content/Community/CommunityModel.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  CommunityModel.swift
+//  Mlem
+//
+//  Created by Sjmarf on 20/09/2023.
+//
+
+import Dependencies
+import Foundation
+
+struct CommunityModel {
+    @Dependency(\.apiClient) private var apiClient
+    @Dependency(\.errorHandler) var errorHandler
+    @Dependency(\.hapticManager) var hapticManager
+    @Dependency(\.communityRepository) var communityRepository
+    
+    enum CommunityError: Error {
+        case noData
+    }
+    
+    @available(*, deprecated, message: "Use attributes of the CommunityModel directly instead.")
+    var community: APICommunity
+    
+    // Ids
+    let communityId: Int
+    let instanceId: Int
+    
```

Comment:
This should also get a `ContentModelIdentifier` id for consistency and to play nice in mixed feeds

Reply:
It's got one in `CommunityModel+ContentModel.swift`